### PR TITLE
mock data for new cars and top cars

### PR DESCRIPTION
**Pull request by setar92**
https://gyazo.com/4d937f9e75a9e30700ca1b384d543151

### CODE REVIEW

File: frontend/src/components/landing-page/landing-page.tsx [hunk 1]
```diff
@@ -57,6 +58,16 @@ export const LandingPage = (): React.ReactElement => {
             {/*<NewCarCard />*/}
             {/*<NewCarCard />*/}
             {/*<NewCarCard />*/}
+            {!newCars
+              ? null
+              : newCars?.map((car) => (
```
Comment (oleksandr-danylchenko) [inline frontend/src/components/landing-page/landing-page.tsx]: Use `&&` condition in React components

File: frontend/src/components/landing-page/mock-new-cars.ts [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  };
+  bodyType: string;
+  manufactureCountry: string;
+  pricesRanges: {
+    price_start: number;
+    price_end: number;
+  }[];
+  description: string;
+  complectationName?: string;
+};
```
Comment (oleksandr-danylchenko) [inline frontend/src/components/landing-page/mock-new-cars.ts]: Don't we already have a type for the model response?
@iryb, could you help me and @setar92 find that if you faced it?

**Review by oleksandr-danylchenko — CHANGES_REQUESTED** [review]

File: frontend/src/components/landing-page/mock-new-cars.ts (hunk 2 above)
Comment (iryb) [reply frontend/src/components/landing-page/mock-new-cars.ts]: @oleksandr-danylchenko @setar92 

Yes, it is in shared folder.

https://github.com/BinaryStudioAcademy/bsa-2022-autoline/blob/develop/shared/src/common/types/preferences/wishlist-response-dto.type.ts

**Comment by setar92** [comment]
1. Some corrections according review. 
2. Added mock data for top cars.
https://gyazo.com/c1d8314337acab9ce467d6da73c1238b

File: frontend/src/components/landing-page/mock-top-cars.ts [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+    name: string;
+    logoUrl: string;
+  };
+  price: number;
+  complectationName?: string;
+  speedometr: number;
+  transmission: string;
+  location: string;
+  fuel: string;
+};
```
Comment (oleksandr-danylchenko) [inline frontend/src/components/landing-page/mock-top-cars.ts:15]: @iryb 
Do we have the for such cars?

Comment (oleksandr-danylchenko) [reply frontend/src/components/landing-page/mock-top-cars.ts:15]: @stanislav-ostr 
I remember you worked on the TopCar card. Do we have a type for the card's card?

Comment (setar92) [reply frontend/src/components/landing-page/mock-top-cars.ts:15]: TopCarCard haven't any props. I created this  type TopCars localy just for correct rending TopCarCard element (using mock data). I can't image all properties which will require this element in future, this is why type TopCars is created here.

Comment (stanislav-ostr) [reply frontend/src/components/landing-page/mock-top-cars.ts:15]: Yes I didn't add any props to TopCarCard

**Pull request merged**